lib/12T/magic/ext4mag.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
```

Comment:
can you format this and other files to match https://peps.python.org/pep-0008/

Reply:
deleted - not needed.